slsDetectorServers/jungfrauDetectorServer/slsDetectorFunctionList.c

```diff
@@ -884,19 +884,30 @@ enum detectorSettings setSettings(enum detectorSettings sett) {
     if (sett == UNINITIALIZED)
         return thisSettings;
 
+    const int specialDynamicDacValues[NSPECIALDACS] =
```

Comment:
Could be moved into the switch statement to create the array we need.

Reply:
Yes, i had incorrectly assumed declarations inside switch statements dont work

Reply:
will fix it